NomadAudio/src/AudioEngine.cpp

```
@@ -27,17 +32,41 @@ namespace {
+            const uint8_t allSoundOff[3] = { static_cast<uint8_t>(0xB0u | ch), 120u, 0u };
+            const uint8_t resetControllers[3] = { static_cast<uint8_t>(0xB0u | ch), 121u, 0u };
+            const uint8_t allNotesOff[3] = { static_cast<uint8_t>(0xB0u | ch), 123u, 0u };
+            buf.addEvent(0, allSoundOff, 3);
+            buf.addEvent(0, resetControllers, 3);
+            buf.addEvent(0, allNotesOff, 3);
+        }
+    }
+}
+
+// Global pointer for singleton access
+static AudioEngine* g_audioEngineInstance = nullptr;
```

Clang (14.0.6)

[warning] variable 'g_audioEngineInstance' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)